docs/index.rst

```diff
@@ -6,81 +6,89 @@
+        :link-type: url
+
+        New to *OpenFE*?  Start here and try it from your browser
+
     .. grid-item-card:: Installing OpenFE
         :img-top: _static/Download.svg
         :text-align: center
         :link: installation
         :link-type: doc
 
-        New to *OpenFE*? Check out our installation guide to get it working on your machine!
-        
+        Check out our installation guide to get it working on your machine!
+
+    .. grid-item-card:: Available Protocols
+        :img-top: _static/CLI.svg
+        :text-align: center
+        :link: guide/protocols/index
+        :link-type: doc
+
+        Documentation on available methods included in the openfe package
+
+    .. grid-item-card:: Changelog
+        :img-top: _static/Showcase.svg
+        :text-align: center
+        :link: CHANGELOG
+        :link-type: doc
+
+        Any notable changes in the package for each released version
+
     .. grid-item-card:: Tutorials
         :img-top: _static/Tutorial.svg
         :text-align: center
         :link: tutorials/index
         :link-type: doc
 
-        Worked through examples of how to use the OpenFE toolkit.
-        
+        Worked through examples of how to use the OpenFE toolkit
+
     .. grid-item-card:: User Guide
         :img-top: _static/UserGuide.svg
         :text-align: center
         :link: guide/index
         :link-type: doc
 
-        Learn about the underlying concepts of the OpenFE toolkit.
-        
-    .. grid-item-card:: API Reference
-        :img-top: _static/API.svg
-        :text-align: center
-        :link: reference/index
-        :link-type: doc
-
-        Get details on the toolkit's core methods and classes.
+        Explanations of the underlying concepts behind the OpenFE toolkit
 
     .. grid-item-card:: Cookbook
         :img-top: _static/Cookbook.svg
         :text-align: center
         :link: cookbook/index
         :link-type: doc
 
-        How-to guides on how to utilise the toolkit components.
-        
-    .. grid-item-card:: Using the CLI
-        :img-top: _static/CLI.svg
-        :text-align: center
-        :link: guide/cli/index
-        :link-type: doc
+        How-to guides for toolkit components
 
-        Reference guide on using the OpenFE CLI.
-
-    .. grid-item-card:: Changelog
-        :img-top: _static/Showcase.svg
-        :text-align: center
-        :link: CHANGELOG
-        :link-type: doc
-
-        Any notable changes in the package for each released version
-        
-    .. grid-item-card:: Relative Free Energy Protocol
-        :img-top: _static/Rocket.svg
+    .. grid-item-card:: API Reference
+        :img-top: _static/API.svg
         :text-align: center
-        :link: reference/api/openmm_rfe
+        :link: reference/index
         :link-type: doc
 
-        Documentation for OpenFE's OpenMM-based Hybrid Topology Relative Free
-        Energy Protocol.
+        Technical details on the toolkit's core methods and classes
 
 
 .. toctree::
```

Comment:
stupid question do we need this here?
```suggestion

```

I think it makes this, right?
![image](https://github.com/OpenFreeEnergy/openfe/assets/12428005/398be625-014e-4814-b85d-4ef2fd4c3d77)

Reply:
seems to create warnings without 
```
checking consistency... /home/richard/code/openfe/docs/CHANGELOG.rst: WARNING: document isn't included in any toctree
/home/richard/code/openfe/docs/cookbook/index.rst: WARNING: document isn't included in any toctree
/home/richard/code/openfe/docs/guide/index.rst: WARNING: document isn't included in any toctree
/home/richard/code/openfe/docs/installation.rst: WARNING: document isn't included in any toctree
/home/richard/code/openfe/docs/reference/index.rst: WARNING: document isn't included in any toctree
/home/richard/code/openfe/docs/tutorials/index.rst: WARNING: document isn't included in any toctree
```